scalableminds/webknossos
# Context action to move tree to group

### Steps to test:
- Go to any annotions view of a dataset and switch to the `Skeleton` tab
- With any tree active, right-click a group and make sure there is a context action called `Move active tree here`
- Repeat this step having selected multiple trees. Make sure the label of the context action is adjusted to "trees".
- Actually move a single and multiple trees to a group and see that it works.
- It is not yet possible to move a group to another group via context action. Thus the context action shouldnt appear if a group is active and you want to move it to another group.
- After having selected multiple trees, clear the selection. Right-click a group and make sure there is no context action called `Move active [group|trees?] here` without an active component.

### Issue:
- fixes #6973 

------
